unified_planning/grpc/unified_planning.proto

```diff
@@ -531,6 +531,31 @@ message Engine {
 }
 
 
+// Message sent by the validator.
+message ValidationResult {
+    enum ValidationResultStatus {
+        // The Plan is valid for the Problem; error_info can't be set.
+        VALID = 0;
+        // The Plan is not valid for the Problem; error_info must be set.
+        INVALID = 1;
+    }
+    ValidationResultStatus status = 1;
+    // Optional. Information about why the Plan is not valid for the Problem
+    string error_info = 2;
```

Comment:
Maybe this could reuse the log message of plan results. I think the 4 verbosity level there are equally applicable to validation output (and we definitly want to allow having detailed feedback from the validator).